tools/readme-catalog/gen-audit-catalog.js

```diff
@@ -0,0 +1,62 @@
+import * as fs from 'fs';
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+import { getGeneratedStringWithReplacement } from './common.js';
+
+/**
+ * Generates a Cloud Audit Log catalog markdown for the README.
+ * @example Example input:
+ * [{
+ *   serviceName: 'workflows.googleapis.com',
```

Comment:
Any reason this isn't actual valid JSON (with double quotes around property names, and double quotes instead of single quotes for values)?

Reply:
Ah, this was taken from JS. Yeah, JSON would be easier. Will change.